app/src/androidTest/assets/pages/basic_nav3.html

```diff
@@ -0,0 +1,10 @@
+<html>
+<head>
+    <script>
+        document.title = Math.random(); // any unique identifier per page load.
```

Comment:
This is effectively going to work as expected, but still bothers me a bit.  Could we change this to `new Date().getTime()`?

Reply:
+1 for getTime().  Forgot much about random() use, but I guess with Java we don't have to set the random seed anymore?